### ISSUE TITLE
Markup not translatable

### ISSUE THREAD

**Issue by Lendude** [Realityloop/markup#3]
Currently the Markup value is not translatable. I know it wasn't in the D7 version yet either (https://www.drupal.org/node/1698102 for reference) but it really should be.

**Comment by Decipher** [comment]
I agree that it should be translatable, but as you point out yourself, it isn't in the D7 version and this is the D7->D8 port. For the sake of not holding up the porting process I will not be implementing this, but it can be tackled once the D8 code is migrated to Drupal.org.
